functions/billing/index.js

```diff
@@ -75,16 +75,11 @@ exports.stopBilling = async (pubsubEvent, context) => {
 /**
  * @return {Promise} Credentials set globally
  */
-const _setAuthCredential = async () => {
-  const res = await auth.getApplicationDefault();
-
-  let client = res.credential;
-  if (client.hasScopes && !client.hasScopes()) {
-    client = client.createScoped([
-      'https://www.googleapis.com/auth/cloud-billing',
-      'https://www.googleapis.com/auth/cloud-platform',
-    ]);
-  }
+const _setAuthCredential = () => {
+  const client = new GoogleAuth({ scopes: [
```

Comment:
I think we should format JSON with normal formatting, like with scopes on a separate line.

Reply:
Done.